src/gui/plugins/video_recorder/VideoRecorder.cc

```diff
@@ -195,17 +194,9 @@ void VideoRecorderPrivate::OnRender()
     if (t - this->updateTime != std::chrono::seconds(0))
     {
       this->updateTime = t;
-      update = true;
     }
   }
```

Comment:
I think we can remove this entire block, I don't think `updateTime` is being used for anything now